include/deal.II/lac/la_parallel_vector.templates.h

```diff
@@ -635,6 +720,47 @@ namespace parallel
 
     template <typename Number>
     void
+    Vector<Number>::import(const ReadWriteVector<Number>                  &V,
+                           VectorOperation::values                         operation,
+                           std_cxx11::shared_ptr<const CommunicationPatternBase> communication_pattern)
+    {
+      // If no communication pattern is given, create one. Otherwise, use the
+      // given one.
+      std_cxx11::shared_ptr<const Utilities::MPI::Partitioner> comm_pattern;
```

Comment:
The only caveat is that with this change we need to release `ReadWriteVector` and `VectorSpaceVector` with the next release. But makes sense anyway.

There is two more thing (I will not manage to do that today), which is to teach ReadWriteVector to get data from a distributed vector and to add functions to copy from and to parallel PETSc/Trilinos vectors. These will be added but marked deprecated because we really want to promote using the ReadWriteVector. My runs revealed that I need to fix the following tests (this is a reminder to myself):
```
	790 - petsc/copy_to_dealvec.mpirun=4.debug (Failed)
	791 - petsc/copy_to_dealvec.mpirun=4.release (Failed)
	1046 - petsc/copy_to_dealvec_block.mpirun=4.debug (Failed)
	1047 - petsc/copy_to_dealvec_block.mpirun=4.release (Failed)
	6626 - mpi/parallel_block_vector_01.mpirun=1.debug (Failed)
	6627 - mpi/parallel_block_vector_01.mpirun=1.release (Failed)
	6668 - mpi/parallel_block_vector_01.mpirun=4.debug (Failed)
	6669 - mpi/parallel_block_vector_01.mpirun=4.release (Failed)
	7434 - mpi/parallel_block_vector_01.mpirun=10.debug (Failed)
	7435 - mpi/parallel_block_vector_01.mpirun=10.release (Failed)
	7632 - multigrid/step-16-04.debug (Failed)
	7633 - multigrid/step-16-04.release (Failed)
	7799 - trilinos/copy_to_dealvec.mpirun=4.debug (Failed)
	7800 - trilinos/copy_to_dealvec.mpirun=4.release (Failed)
	8143 - trilinos/copy_to_dealvec_block.mpirun=4.debug (Failed)
	8144 - trilinos/copy_to_dealvec_block.mpirun=4.release (Failed)
```
The problem in the step-16 test is that I do not have an operation `LA::distributed::Vector::add(Vector)` but only the += operation.